arches/app/models/graph.py

```diff
@@ -724,6 +724,13 @@ def save(self, validate=True, nodeid=None):
         return self
 
     def delete(self):
+        self.delete_associated_entities()
+        super(Graph, self).delete()
```

Comment:
While here, would you mind returning this value instead of swallowing it? #preexistingissue